kubejs/startup_scripts/script.js

```diff
@@ -39,7 +39,7 @@ onEvent('item.registry', function (event) {
     food.eaten(function (e) {
       if (!e.player.stages.has('end_access')) {
         e.player.stages.add('end_access');
-        e.player.tell(text.green("You have unlocked access to The End"));
+        e.player.tell(text.green("您已解锁访问末地"));
```

Comment:
https://minecraft.fandom.com/zh/wiki/%E6%9C%AB%E5%9C%B0?variant=zh
```suggestion
        e.player.tell(text.green("您已解锁前往末地的权限"));
```